README.md

```diff
@@ -91,25 +99,37 @@ observability.tracings.output.enabled=false
 ### Step 3
 Clone demo source code and build
 ```
-git clone https://github.com/akwei/spring-petclinic-microservices.git
-cd spring-petclinic-microservices
-mvn -DskipTests=true package
+$ git clone https://github.com/akwei/spring-petclinic-microservices.git
+$ cd spring-petclinic-microservices
+$ mvn -DskipTests=true package
 ```
 
 ### Step 4
 Run java application with agent in console
 ```
-java -jar spring-petclinic-config-server/target/spring-petclinic-config-server-2.4.2.jar
+# Open another console
+$ java -jar spring-petclinic-config-server/target/spring-petclinic-config-server-2.4.2.jar
 
-java -jar spring-petclinic-discovery-server/target/spring-petclinic-discovery-server-2.4.2.jar
+# Open another console
+$ java -jar spring-petclinic-discovery-server/target/spring-petclinic-discovery-server-2.4.2.jar
 
-java -javaagent:/[user path]/easeagent.jar=/{path}/agent.properties -Deaseagent.server.port=9900 -jar spring-petclinic-vets-service/target/spring-petclinic-vets-service-2.4.2.jar
+# Open another console
+$ export EASE_AGENT_PATH=[Replace with agent path]
+$ java -javaagent:${EASE_AGENT_PATH}/easeagent-dep.jar=${EASE_AGENT_PATH}/agent.properties -Deaseagent.server.port=9900 -jar spring-petclinic-vets-service/target/spring-petclinic-vets-service-2.4.2.jar
```

Comment:
it's better to add the quote for "-javaagent: ... ... ", just in case there might be space in the directory path.